docs/howtoguides/how_to_use_manifest_file_for_pro_vulnerability.rst

```diff
@@ -0,0 +1,25 @@
+How to provide a manifest file to pro vulnerability commands
+************************************************************
+
+If you want to provide a manifest file to either ``pro vulnerability show`` or
```

Comment:
Why would someone want to do that?

Reply:
You might want the see the CVE status for a different machine than the one you are using. The manifest file is the way to achieve that

Reply:
Can we mention that here as an example use case? (I'm not sure it's that obvious)